modules/core/src/main/scala/ru/tinkoff/phobos/decoding/AttributeDecoder.scala

```diff
@@ -136,6 +136,9 @@ object AttributeDecoder extends AttributeLiteralInstances {
   implicit val zonedDateTimeDecoder: AttributeDecoder[ZonedDateTime] =
     stringDecoder.emap(wrapException(ZonedDateTime.parse))
 
+  implicit val offsetDateTimeDecoder: AttributeDecoder[OffsetDateTime] =
```

Comment:
I think, that encoding of `OffsetDateTime` should be implemented via `OffsetDateTime.toString` and decoding - via `OffsetDateTime.parse`, like it currently is for  `LocalDateTime`, `ZonedDateTime` etc.

En(De)coding via `zonedDateTimeEn(De)coder` seems to work almost the same way as `.toString`/`.parse`. The only exception - current implementation will parse `2022-02-20T20:27:29.771+03:00[Europe/Moscow]` as `OffsetDateTime`, but `.parse` will throw an exception. It seems to be non-obvious, and I believe that decoder should work the same way as `.parse` does. If current behavior is desired by user, it may be implemented explicitly

Reply:
Oh, this is fair.
I have changed implementations - now they use `stringDecoder `composition.